test/unit/go.mod

```diff
@@ -0,0 +1,78 @@
+module github.com/aws/cluster-api-provider-cloudstack-staging/test/unit
+
+go 1.17
+
+require (
+	github.com/apache/cloudstack-go/v2 v2.13.0
+	github.com/aws/cluster-api-provider-cloudstack v0.4.3
```

Comment:
we're past v0.4.3 now, it's v0.4.5-rc3 I believe. But I guess we're replacing it anyway below? Could you explain the reasoning behind this?